### PR TITLE
Farber/pcx 7037 registrar updates

### PR DESCRIPTION
**Pull request by jason-cf**
Re-structuring and updating content

### CODE REVIEW

File: content/_redirects [hunk 1]
```diff
@@ -562,6 +562,10 @@
 /registrar/transfer-instructions/namecheap/ /registrar/get-started/transfer-domain-to-cloudflare/ 301
 /registrar/transfer-instructions/oneandone/ /registrar/get-started/transfer-domain-to-cloudflare/ 301
 /registrar/why-choose-cloudflare/whois-redaction/ /registrar/get-started/whois-redaction/ 301
+/registrar/reference/custom-domain-protection/ registrar/custom-domain-protection/ 301
+
+# Registrar FAQ
+/registrar/faq/#can-i-register-a-premium-domain /registrar/get-started/register-domain/ 301
```
Comment (marciocloudflare) [inline content/_redirects]: We can't create redirects with a `#` on the source URL, just the destination

File: content/registrar/_partials/_requirements-simple.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+All domains using Cloudflare Registrar must use Cloudflare DNS on a [full setup](/dns/zone-setups/full-setup/)[^1]. This also means that you cannot change to another DNS provider while using Cloudflare Registrar.
```
Comment (marciocloudflare) [inline content/registrar/_partials/_requirements-simple.md]: That second sentence is a bit weird bc the subject (domains) is far, far away, in the beginning of the other sentence. My suggestion is to say it like so: `This also means that you cannot move your domain to another DNS provider while using Cloudflare Registrar.` - or something similar.

File: content/_redirects (hunk 1 above)
Comment (marciocloudflare) [reply content/_redirects]: Also, do you have all the redirects you need? The build errored out bc of wrong links

**Review by marciocloudflare — APPROVED** [review]

File: content/_redirects (hunk 1 above)
Comment (jason-cf) [reply content/_redirects]: Ah okay. This is seemingly what is causing the build to fail.

**Pull request merged**